wheel 0.35.0 on FreeBSD fails

It seems using 0.35.0 caused an error in our CI pipeline on FreeBSD:
```
16:48:05             File "/tmp/pip-build-env-3dsgn9h4/overlay/lib/python3.7/site-packages/wheel/bdist_wheel.py", line 328, in run
16:48:05               impl_tag, abi_tag, plat_tag = self.get_tag()
16:48:05             File "/tmp/pip-build-env-3dsgn9h4/overlay/lib/python3.7/site-packages/wheel/bdist_wheel.py", line 278, in get_tag
16:48:05               assert tag in supported_tags, "would build wheel with unsupported tag {}".format(tag)
16:48:05           AssertionError: would build wheel with unsupported tag ('cp37', 'cp37m', 'freebsd_12_1_RELEASE_amd64')
16:48:05           ----------------------------------------
16:48:05           ERROR: Failed building wheel for bcrypt
```
could you advice what's going on there and how we could fix it?

--- a/docs/news.rst
+++ b/docs/news.rst
@@ -1,6 +1,11 @@
 Release Notes
 =============
 
+**UNRELEASED**
+
+- Fixed ``bdist_wheel`` not working on FreeBSD due to mismatching platform tag
+  name (it was not being converted to lowercase)
+
 **0.35.0 (2020-08-13)**
 
 - Switched to the packaging_ library for computing wheel tags

--- a/src/wheel/bdist_wheel.py
+++ b/src/wheel/bdist_wheel.py
@@ -254,7 +254,7 @@ class bdist_wheel(Command):
             if plat_name in ('linux-x86_64', 'linux_x86_64') and sys.maxsize == 2147483647:
                 plat_name = 'linux_i686'
 
-        plat_name = plat_name.replace('-', '_').replace('.', '_')
+        plat_name = plat_name.lower().replace('-', '_').replace('.', '_')
 
         if self.root_is_pure:
             if self.universal:

